sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidConvention.java

```diff
@@ -73,7 +73,7 @@ public RelTraitDef getTraitDef()
   @Override
   public boolean satisfies(RelTrait trait)
   {
-    return trait == this;
+    return trait.equals(this);
```

Comment:
Do we need this change? This class is singleton and equals() is not implemented for this class.

Reply:
It's ok, IMO.